.pa11yci

```diff
@@ -0,0 +1,14 @@
+{
+  "urls": [
+    "http://localhost:8080",
+    "http://localhost:8080/audit/new/"
+  ],
+  "defaults": {
+    "standard": "WCAG2AA",
+    "runners": [
+      "htmlcs",
+      "axe"
+    ],
+    "hideElements": ".sign-in-logo span, header button span, .usa-step-indicator__segment-label"
```

Comment:
I've checked these manually and they're all fine. The only potentially iffy one is the "login.gov" text in the login button, but the contrast there is only questionable for longer pieces of text.